python/py-rawpy/files/setup.py.diff

```diff
@@ -0,0 +1,13 @@
+--- setup.orig.py	2022-10-14 21:20:34.000000000 +0200
++++ setup.py	2022-10-14 21:21:11.000000000 +0200
+@@ -23,8 +23,8 @@
+ buildGPLCode = os.getenv('RAWPY_BUILD_GPL_CODE') == '1'
+ 
+ # don't treat mingw as Windows (https://stackoverflow.com/a/51200002)
+-isWindows = os.name == 'nt' and 'GCC' not in sys.version
+-isMac = sys.platform == 'darwin'
++isWindows = False
++isMac = False
```

Comment:
why would this need to be false?

Reply:
Because otherwise they build their own libraw. I plan to contact upstream and propose to add a flag instead, but it will be easier to argue that it is useful if I can show them that MacPorts is a viable alternative for the installation.

Reply:
To be clearer, they assume that linux has a libraw package available, hence the difference in behavior. Their conditionnals are ill-named. Mac=True implies to build libraw and change how to link to it. Mac=Windows=False implies Linux to them, and therefore normal building and linking process, which is what we want.

Reply:
okay - thanks! I must admit that I didn't look any further than the lines in the patchfile to see what's going on - it just seemed a little strange to me ;)